src/SFA.DAS.EAS.Web/Views/TransferConnectionInvitations/Index.cshtml

```diff
@@ -8,6 +10,21 @@
 <div class="grid-row">
     <div class="column-two-thirds">
         <h1 class="heading-xlarge">Connect with an employer</h1>
+
+        @if (Model.IsPendingReceiver)
```

Comment:
I think this logic is incorrect, should it be using `IsReceiver`?

Reply:
So it should